Doc/library/unittest.rst

```diff
@@ -139,12 +139,12 @@ line, the above script produces an output that looks like this::
 Passing the ``-v`` option to your test script will instruct :func:`unittest.main`
 to enable a higher level of verbosity, and produce the following output::
 
-   test_isupper (__main__.TestStringMethods) ... ok
-   test_split (__main__.TestStringMethods) ... ok
-   test_upper (__main__.TestStringMethods) ... ok
+   test_isupper (__main__.TestStringMethods.test_isupper) ... ok
+   test_split (__main__.TestStringMethods.test_split) ... ok
+   test_upper (__main__.TestStringMethods.test_upper) ... ok
 
    ----------------------------------------------------------------------
-   Ran 3 tests in 0.001s
+   Ran 3 tests in 0.000s
```

Comment:
```suggestion
   Ran 3 tests in 0.001s
```
Leave a small execution time even if your machine is super-fast.
It removes unnecessary user surprising.